src/operator/audit.ts

```diff
@@ -86,6 +86,9 @@ class AuditSubscriber<T, R> extends OuterSubscriber<T, R> {
         this.destination.error(errorObject.e);
       } else {
         this.add(this.throttled = subscribeToResult(this, duration));
+        if (this.throttled.closed) {
```

Comment:
I have assumed this change's similar to this -> https://github.com/ReactiveX/rxjs/pull/1490, where there are synchronous observables are supplied `subscribeToResult` completes immediately instead of scheduled.